doc/source/conf.py

```diff
@@ -162,7 +162,7 @@ def parse_schema(d, md=[], depth=0, pre=""):
         depth += 1
         # Create markdown headers for each schema level
         for key, val in d["properties"].items():
-            md.append("(schema:%s)=" % (pre + key))
+            md.append("(schema_%s)=" % (pre + key))
```

Comment:
@choldgraf I think there may be one or two issues to report in the MyST ecosystem, do you have suggestions on what repo's to report this in?

Reply:
The MyST parser repo or the Jupyter book repo both work!